docs-site/content/en/db/project-db-directory.md

```
@@ -0,0 +1,319 @@
+
+### Full Rebuild
+
+For full rebuild when needed:
+
+```bash
+/moai db refresh
+```
+
+Prompt:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)